dockerfiles/centos_7

```diff
@@ -97,9 +96,11 @@ repo_gpgcheck = 1\n'\
         perl \
     && rm -rf /var/cache/yum \
     && : "---------- gosu ----------" \
+    && if [[ $(uname -m) = 'x86_64' ]]; then GOSU_ARCH='amd64'; fi \
```

Comment:
This part looks great. How did it work before?

Reply:
It didn't work before, I really don't know how was built images before with this bag